cpp/include/nvstrings/NVText.h

```diff
@@ -55,6 +55,23 @@ class NVText
      */
     static NVStrings* unique_tokens(NVStrings& strs, const char* delimiter=nullptr);
 
+    /**
+     * @brief Tokenize all input strings into single characters strings.
+     * This will create single character strings in the returned instance.
```

Comment:
@brief doesn't end until the end of a paragraph, so always put a blank line after your brief.
http://www.doxygen.nl/manual/commands.html#cmdbrief

```suggestion
     *
     * Creates single-character strings in the returned instance.
```

Reply:
Actually this line may be redundant with the brief.